### PR TITLE
Update run_cupy_test.py

### PR DESCRIPTION
**Pull request by mitmul**
Add pynvrtc to requires for https://github.com/cupy/cupy/pull/24

### CODE REVIEW

**Comment by unnonouno** [comment]
Fix setup.py instead of adding requirement here. And send the fix to _v2 branch.

File: run_cupy_test.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
     if args.test == 'py2':
         conf = {
             'base': 'ubuntu14_py2',
-            'cuda': 'cuda65',
+            'cuda': 'cuda70',
```
Comment (unnonouno) [inline run_cupy_test.py]: Use 'cuda75' for 'py3' test.

Comment (mitmul) [inline run_cupy_test.py]: Thanks. Fixed

**Comment by okuta** [comment]
This problem fixed by #232.

**Pull request closed without merging**